<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QMainWindow" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="styleSheet">
       <string notr="true">QFrame#mainFrame{
	border-radius: 20px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="topBar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(35, 35, 35);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_toggle">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 67, 153, 255), stop:1 rgba(136, 37, 147, 255));</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="instagramLogo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>assets/ig_logo.png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_top">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="usernameLbl">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>681</width>
               <height>40</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Gotham Pro Black</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background-color: rgb(35,35,35);</string>
             </property>
             <property name="text">
              <string>username</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QPushButton" name="closeBtn">
             <property name="geometry">
              <rect>
               <x>684</x>
               <y>12</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
background-color: rgb(255, 66, 19);
border-radius: 8px;
}

QPushButton:hover{
	background-color: rgb(255, 66, 19,200);

}

</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="minimizeBtn">
             <property name="geometry">
              <rect>
               <x>660</x>
               <y>12</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
	background-color: rgb(255, 255, 127,200);
}
QPushButton{
background-color: rgb(255, 255, 127);
border-radius: 8px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="dropShadowFrame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_left_menu">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(35, 35, 35);
border-radius: 10px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QFrame" name="frame_top_menus">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="likeBtnSection">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Gotham Pro</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(67, 102, 201, 255), stop:0.517045 rgba(186, 49, 181, 255), stop:1 rgba(230, 71, 110, 255));
}</string>
                  </property>
                  <property name="text">
                   <string>Like</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="commentBtnSection">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Gotham Pro</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(67, 102, 201, 255), stop:0.517045 rgba(186, 49, 181, 255), stop:1 rgba(230, 71, 110, 255));
}</string>
                  </property>
                  <property name="text">
                   <string>Comment</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="followBtnSection">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Gotham Pro</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(67, 102, 201, 255), stop:0.517045 rgba(186, 49, 181, 255), stop:1 rgba(230, 71, 110, 255));
}</string>
                  </property>
                  <property name="text">
                   <string>Follow</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="messageBtnSection">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Gotham Pro</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(67, 102, 201, 255), stop:0.517045 rgba(186, 49, 181, 255), stop:1 rgba(230, 71, 110, 255));
}</string>
                  </property>
                  <property name="text">
                   <string>Message</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_pages">
            <property name="styleSheet">
             <string notr="true">QFrame{
	border-radius:10px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="likeSection">
                <widget class="QLineEdit" name="linkPostTxt">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>60</y>
                   <width>660</width>
                   <height>40</height>
                  </rect>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Gotham Pro</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);</string>
                 </property>
                </widget>
                <widget class="QLabel" name="postLinkLbl">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>10</y>
                   <width>120</width>
                   <height>45</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>24</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #FFF;
background-color: rgb(35, 35, 35);
BORDER-RADIUS: 0PX;</string>
                 </property>
                 <property name="text">
                  <string>Post link</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
                <widget class="QPushButton" name="likeBtn">
                 <property name="geometry">
                  <rect>
                   <x>300</x>
                   <y>110</y>
                   <width>101</width>
                   <height>30</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Gotham Pro Black</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #FFF;
border-radius: 15px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(67, 102, 201, 255), stop:0.517045 rgba(186, 49, 181, 255), stop:1 rgba(230, 71, 110, 255));</string>
                 </property>
                 <property name="text">
                  <string>LIKE!</string>
                 </property>
                </widget>
                <widget class="QLabel" name="chooseFileLbl">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>200</y>
                   <width>541</width>
                   <height>45</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>24</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #FFF;
background-color: rgb(35, 35, 35);
BORDER-RADIUS: 0PX;</string>
                 </property>
                 <property name="text">
                  <string>Choose a txt file (multiple posts likes)</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
                <widget class="QPushButton" name="chooseBtn">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>260</y>
                   <width>141</width>
                   <height>31</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Gotham Pro Black</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #FFF;
border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(67, 102, 201, 255), stop:0.517045 rgba(186, 49, 181, 255), stop:1 rgba(230, 71, 110, 255));</string>
                 </property>
                 <property name="text">
                  <string>CHOOSE FILE</string>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_2">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="font">
                    <font>
                     <pointsize>40</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #FFF;</string>
                   </property>
                   <property name="text">
                    <string>PAGE 2</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_3">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <pointsize>40</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #FFF;</string>
                   </property>
                   <property name="text">
                    <string>PAGE 3</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
